arch/arm/src/imx6/imx_enet.c

```diff
@@ -2252,16 +2285,13 @@ static inline int imx_initphy(struct imx_driver_s *priv, bool renogphy)
    * configuration and the auto negotiation results.
    */
 
-#ifdef CONFIG_IMX_ENETUSEMII
   rcr = ENET_RCR_CRCFWD |
         (CONFIG_NET_ETH_PKTSIZE + CONFIG_NET_GUARDSIZE)
           << ENET_RCR_MAX_FL_SHIFT |
+#ifdef CONFIG_IMX_ENETUSEMII
         ENET_RCR_MII_MODE;
 #else
-  rcr = ENET_RCR_RMII_MODE | ENET_RCR_CRCFWD |
-        (CONFIG_NET_ETH_PKTSIZE + CONFIG_NET_GUARDSIZE)
-          << ENET_RCR_MAX_FL_SHIFT |
-        ENET_RCR_MII_MODE;
+        ENET_RCR_RGMII_EN;
```

Comment:
Why `ENET_RCR_RMII_MODE` is dropped?

Reply:
@pkarashchenko 

>Why ENET_RCR_RMII_MODE is dropped?

Because the reference manual says that we do not set both.